WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -331,8 +332,43 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: ListTableViewCell.defaultReuseID, for: indexPath)
-        configureCell(cell, at: indexPath)
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: NotificationsTableViewCellContent.reuseIdentifier) as? HostingTableViewCell<NotificationsTableViewCellContent>,
+              let note = tableViewHandler.resultsController?.managedObject(atUnsafe: indexPath) as? Notification else {
+            return UITableViewCell()
+        }
+        cell.selectionStyle = .none
+        if let deletionRequest = notificationDeletionRequests[note.objectID] {
+            cell.host(
+                NotificationsTableViewCellContent(
+                    style: .altered(
+                        .init(
+                            text: deletionRequest.kind.legendText,
+                            action: { [weak self] in
+                                self?.cancelDeletionRequestForNoteWithID(note.objectID)
+                            }
+                        )
+                    )
+                ),
+                parent: self
+            )
+        } else {
+            cell.host(
+                NotificationsTableViewCellContent(
+                    style: .regular(
+                        .init(
+                            title: note.renderSubject()?.string ?? "",
+                            description: note.renderSnippet()?.string,
+                            shouldShowIndicator: !note.read,
+                            avatarStyle: .init(urls: note.allAvatarURLs) ?? .single(note.iconURL!),
```

Comment:
```
 .single(note.iconURL!)
```

[Suggestion] Honestly I wouldn't be confident force unwrapping a value that comes from the Backend.  In my opinion,  the `avatarStyle` can be `nil`, and in that case, we should display some placeholder.

Reply:
Resolved https://github.com/wordpress-mobile/WordPress-iOS/pull/22593/commits/9abb1cc8615c3f548f4c14d98b6a37179e90fda0